src/Controllers/AdminController.php

```
@@ -778,6 +801,117 @@ public function billing(): void
+
+            return $this->emptyAiUsageSummary($includedCredits, $periodStart, $periodEnd, false);
+        }
+    }
+
+    /**
+     * @param array<string, mixed>|null $subscription
+     * @return array{0: \DateTimeImmutable, 1: \DateTimeImmutable, 2: \DateTimeImmutable}
+     */
+    private function currentBillingPeriod(?array $subscription): array
+    {
+        $tz = new \DateTimeZone('UTC');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '847', column '19'). (undefined)

(MissingImport)